### PR TITLE
Fix doc and auth issue

### LINKED ISSUE #139: downloader fails
Traceback (most recent call last):
  File "/Users/chhu02/anaconda3/bin/icloudpd", line 10, in <module>
    sys.exit(main())
  File "/Users/chhu02/anaconda3/lib/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/Users/chhu02/anaconda3/lib/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/Users/chhu02/anaconda3/lib/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/chhu02/anaconda3/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/Users/chhu02/anaconda3/lib/python3.6/site-packages/icloudpd/base.py", line 229, in main
    client_id=os.environ.get("CLIENT_ID"),
  File "/Users/chhu02/anaconda3/lib/python3.6/site-packages/icloudpd/authentication.py", line 32, in authenticate
    client_id=client_id)
  File "/Users/chhu02/anaconda3/lib/python3.6/site-packages/pyicloud_ipd/base.py", line 193, in __init__
    self.authenticate()
  File "/Users/chhu02/anaconda3/lib/python3.6/site-packages/pyicloud_ipd/base.py", line 216, in authenticate
    raise PyiCloudFailedLoginException(msg, error)
pyicloud_ipd.exceptions.PyiCloudFailedLoginException: ('Invalid email/password combination.', PyiCloudAPIResponseError('Failed to validate the credentials from cookie',))

### PR DESCRIPTION
**Pull request by andersjohansson**
Refer to newer pyicloud-ipd (see https://github.com/ndbroadbent/pyicloud/pull/3).
Include fix from @danielfmolnar (don’t really know what that does).
Document changed command line options.

Fixes #139

### CODE REVIEW

**Comment by apetresc** [comment]
Where is `pyicloud_ipd==0.11.0` coming from? It doesn't appear to be on PyPI and I can't find the source repo.

**Comment by apetresc** [comment]
Oh, I see, it's probably [this PR right here](https://github.com/ndbroadbent/pyicloud/pull/3/files).

File: icloudpd/base.py [hunk 1]
```diff
@@ -350,17 +350,17 @@ def photos_exception_handler(ex, retries):
...
-                    "Skipping %s, only downloading photos." % photo.filename
-                )
-                break
-            if photo.item_type != "image" and photo.item_type != "movie":
-                logger.set_tqdm_description(
-                    "Skipping %s, only downloading photos and videos. "
-                    "(Item type was: %s)" % (photo.filename, photo.item_type)
-                )
-                break
+            # if skip_videos and photo.item_type != "image":
```
Comment (octplane) [inline icloudpd/base.py:353]: If this is really causing errors, can you provide some stacktrace or insight of what's breaking right now? Also, this should be a separate pull request and remove the code completely. Commenting out broken out will give people the impression that it could work, when in fact it's broken...

**Comment by ndbroadbent** [comment]
Hello, thanks for the PR, and sorry for the delay! Unfortunately I don't have time to investigate and test the issue, but if you are still interested, could you (or someone else) please rebase and fix the conflicts, and then make sure all the tests and linters are passing? Thanks a lot!

**Review by menkej — COMMENTED** [review]
I think this pull request is obsolete. The README is already fixed by some other change. The commented out code block should not be disabled as it is a working feature. Boosting the required lib versions might be a good thing later though. I vote for rejecting this now. I hope you're ok with it @andersjohansson .

**Comment by menkej** [comment]
Documentation issues have been addressed in #197, all the other issues should be fixed already for quite some time. If you still experience these issues, please open a new ticket! Thanks!
